### ISSUE TITLE
Request for API IP Address Whitelist to Bypass Server Access Restrictions

### ISSUE THREAD

**Issue by strakadavid** [allegro/allegro-api#9951]
Hi,

could you provide me with a list of IP addresses that your API uses to download images from the provided URLs? We have restricted access on our server, and your API is hitting the limits we have set. I would like to whitelist your IP addresses. Thank you.

**Comment by Lukasz-Zurek** [comment]
Yes. Please use this [contact form](https://allegro.pl/pomoc/kontakt) and add this Github thread's case number #9951.
